test/land/create_bkg_ens.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -e
+################################################
+# 4. CREATE BACKGROUND ENSEMBLE (LETKFOI)
+################################################
+YY=2021
+MM=03
+DD=23
+HH=18
+HR=12
+FILEDATE=$YY$MM$DD.${HH}0000
+
+project_binary_dir=$1
+project_source_dir=$2
+
+GYMD=$(date +%Y%m%d -d "$YY$MM$DD $HH - 6 hours")
+GHR=$(date +%H -d "$YY$MM$DD $HH - 6 hours")
+
+WORKDIR=$project_binary_dir/test/testrun
+RSTDIR=$GDASAPP_TESTDATA/lowres/gdas.$GYMD/$GHR/atmos/RESTART/
```

Comment:
@jiaruidong2017 , where is `GDASAPP_TESTDATA` defined? From the log you pasted, it looks like that variable is empty.

Reply:
never mind, it's defined above in the ua file .... You probably need to pass it as part of the "environment" for your ctest.

Reply:
@guillaumevernieres GDASAPP_TESTDATA is defined in the "modulefiles/GDAS/hera.lua" as: 

`setenv("GDASAPP_TESTDATA","/scratch1/NCEPDEV/da/Cory.R.Martin/CI/GDASApp/data")`